src/components/header/headlines.scss

```
@@ -0,0 +1,55 @@
+$black-color: #000000;
+$white-color: #ffffff;
+
+.container {
+  margin-right: auto;
+  margin-left: auto;
```

Better to use like this
```suggestion
  margin: 0 auto;
```